src/frontend/contexts/products-context.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect, createContext, useContext } from "react";
+const { default: axios } = require("axios");
```

Comment:
use the basic one, dont use this as it is specific to node, its not wrong but still not needed

Reply:
yup ! i made the change would add that in the next PR